urdf/sdf_extensions.md

```diff
@@ -0,0 +1,822 @@
+      <a href="http://www.ode.org">Open Dynamics Engine (ODE)</a>
+      (see parameter descriptions in <a href="http://www.ode.org/ode-latest-userguide.html#sec_7_3_7">ODE's user guide</a>)
+    </td>
+    <td><code>collision/surface/friction/ode/mu</code></td>
+  </tr>
+  <tr>
+    <td>mu2</td>
+    <td><code>collision/surface/friction/ode/mu2</code></td>
+  </tr>
+  <tr>
+    <td>fdir1</td>
+    <td>vector</td>
+    <td>3-tuple specifying direction of mu1 in the collision local reference frame.</td>
+    <td><code>collision/surface/friction/ode/fdir1</code></td>
+  </tr>
+  <tr>
+    <td>kp</td>
+    <td rowspan="2">double</td>
+    <td rowspan="2">Contact stiffness k_p and damping k_d for rigid body contacts as defined by ODE
+      (<a href="http://www.ode.org/ode-latest-userguide.html#sec_7_3_7">ODE uses erp and cfm</a>
+      but there is a
+      <a href="https://github.com/osrf/gazebo/blob/gazebo9/gazebo/physics/ode/ODEJoint.cc">mapping between erp/cfm and stiffness/damping</a>)
+    </td>
+    <td><code>collision/surface/contact/ode/kp</code></td>
+  </tr>
+  <tr>
+    <td>kd</td>
+    <td><code>collision/surface/contact/ode/kd</code></td>
+  </tr>
+  <tr>
+    <td>selfCollide</td>
+    <td>bool</td>
+    <td>If true, the link can collide with other links in the model.</td>
+    <td><code>self_collide</code></td>
+  </tr>
+  <tr>
+    <td>maxContacts</td>
+    <td>int</td>
+    <td>Maximum number of contacts allowed between two entities. This value overrides the max_contacts element defined in physics.</td>
+    <td><code>collision/max_contacts</code></td>
+  </tr>
+  <tr>
+    <td>laserRetro</td>
+    <td>double</td>
+    <td>intensity value returned by laser sensor.</td>
+    <td><code>collision/laser_retro</code></td>
+  </tr>
+  <tr>
+    <td>visual</td>
+    <td>element</td>
+    <td>The content of the element will be inserted into each <code>visual</code> of the SDFormat link</td>
+    <td><code>visual</code></td>
+  </tr>
+  <tr>
+    <td>material</td>
+    <td>element</td>
+    <td>The content of the element will be inserted into each <code>material</code> of the SDFormat link</td>
+    <td><code>visual/material</code></td>
+  </tr>
+  <tr>
+    <td>collision</td>
+    <td>element</td>
+    <td>The content of the element will be inserted into each <code>collision</code> of the SDFormat link</td>
+    <td><code>collision</code></td>
+  </tr>
+</table>
+
+> **Note:** The XPath used in the "Corresponding SDFormat element" column is relative to the link element.
+
+**Example:**
+
+The following shows how to set the the first coefficient of friction for all
+`<collision>` elements in a link
+
+```xml
+<!--URDF-->
+<robot name='friction_example'>
+  <link name='base_link'>
+    <inertial>
+      <mass value='0.12' />
+      <inertia ixx='0.01' ixy='0' ixz='0' iyy='0.01' iyz='0' izz='0.01' />
+    </inertial>
+    <collision>
```

Comment:
add another collision to show that all collisions get the parameter?

Reply:
Done in https://github.com/gazebosim/sdf_tutorials/pull/88/commits/3beee03fc26bf4c00d3002e39380d7672c7351ca